aws/logs_monitoring/lambda_function.py

```diff
@@ -465,6 +486,48 @@ def split(events):
     return metrics, logs
 
 
+# should only be called when INCLUDE_AT_MATCH and/or EXCLUDE_AT_MATCH exist
+def filter_logs(logs):
```

Comment:
I think you could simplify this method by iterating over all logs and for each log, try include, try exclude, this would limit the cases to two.

Reply:
The question now is which rule wins where the log should both included and excluded, on the agent we decided to exclude it https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/processor/processor.go#L78

Reply:
Exclude 🛑 wins here too 👍

Simplified it a bit.